.github/workflows/markdownlint.yml

```diff
@@ -0,0 +1,21 @@
+name: Markdownlint
+
+on: pull_request
```

Comment:
You can filter this with the "paths" so it only runs if affected files are touched

Reply:
Is it worth it, considering that we tell markdownlint what to exclude anyway below? If this would be just about not running the check at all for EF6 changes...